src/components/ResourceList/components/FilterControl/FilterControl.tsx

```diff
@@ -53,6 +55,12 @@ export class FilterControl extends React.Component<CombinedProps> {
       },
     );
 
+    const placeholderLabel = placeholder
```

Comment:
I would think it's probably best that when a custom placeHolderLabel is provided, the consumer of in responsible for passing the translation.

```
const textFieldLabel = placeholder 
 ? placeholder 
 : intl.translate(
      'Polaris.ResourceList.FilterControl.textFieldLabel',
      {
        resourceNamePlural: resourceName.plural.toLocaleLowerCase(),
      },
    );
```

Reply:
Are you thinking that instead of translating the `Search ` part we can just leave the transitional completely up to the consumer? That sounds like a good idea!

Reply:
Yeah, either they use the default we have today, nothing gets changed there but if they do provide a placeholder text it gets used for both the label and the placeholder (because in thsi case they would be the same) as provided by the consumer.